packages/alfa-table/src/table.ts

```diff
@@ -224,29 +270,116 @@ export namespace Table {
               height: Math.max(this.height, this.height + rowGroup.height),
               width: Math.max(this.width, rowGroup.width),
               // merge in new cells
-              cells: this._cells.concat(...rowGroup.cells),
+              cells: this._cells.concat(rowGroup.cells),
               // add new group
-              rowGroups: [...this.rowGroups].concat(rowGroup.rowgroup),
+              rowGroups: List.from(this.rowGroups).append(rowGroup.rowgroup),
             });
           } else {
             return this;
           }
         });
     }
 
+    public hasDataCellCoveringArea(
+      x: number,
+      y: number,
+      w: number,
+      h: number
+    ): boolean {
+      return some(this.cells, (cell) => cell.isDataCoveringArea(x, y, w, h));
+    }
+
+    public addHeadersVariants(): Builder {
+      return this.update({
+        cells: List.from(
```

Comment:
Somehow, removing this `List.from` is breaking all tests. I have little clue why. This is a bit weird and worrying, but I have already spend too much time pointlessly on this…
(it breaks the implicit header detection)